src/store/index.js

```diff
@@ -1,16 +1,33 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 
+import api from "@/api/FactoryRepository.js"
+const moviesApi = api.get("movies")
+
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
+    movies: [],
   },
   getters: {
+    getMovies(state) {
+      return state.movies
+    },
+    getThreePremieres(state) {
```

Comment:
nice that you've created a separate getter for it. In the future it may be easily replaced with proper data when API is ready 👏